src/Nox.Types.EntityFramework/CountryCode3Converter.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace Nox.Types.EntityFramework;
+
+/// <summary>
+///   Class for two-letter country codes (ISO alpha-2).
+/// </summary>
+public class CountryCode3Converter : ValueConverter<CountryCode3,string>
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="CountryCode3Converter" /> class.
+    /// </summary>
+    public CountryCode3Converter() : base (countryCode3 => countryCode3.Value, countryCode3 => CountryCode3.From(countryCode3)) { }
```

Comment:
So its easier to read 
```suggestion
    public CountryCode3Converter() : base (countryCode3 => countryCode3.Value, countryCode3Value => CountryCode3.From(countryCode3Value)) { }
```